app/labs/edit/route.js

```diff
@@ -11,13 +11,22 @@ export default AbstractEditRoute.extend(AddToPatientRoute, ChargeRoute, PatientL
   modelName: 'lab',
   newTitle: t('labs.newTitle'),
   pricingCategory: 'Lab',
+  customForms: Ember.inject.service(),
 
   actions: {
     returnToAllItems() {
       this.controller.send('returnToAllItems');
     }
   },
 
+  getCustomFormData() {
```

Comment:
Take a look at how visit does this:
https://github.com/HospitalRun/hospitalrun-frontend/blob/master/app/visits/edit/route.js#L35-L43

Basically what you need to do is to create a newData object and make sure it includes a custom forms object and then you return  customForms.setDefaultCustomForms(['lab'], newData) from the getNewData function.